src/splitInteger.test.js

```diff
@@ -4,18 +4,18 @@ const splitInteger = require('./splitInteger');
 
 test(`should split a number into equal parts
   if a value is divisible by a numberOfParts`, () => {
-
```

Comment:
Please add a test case description in the test function as a string to make it more readable and understandable. For example: `test('should split a number into equal parts if a value is divisible by a numberOfParts', () => {...})`